website/docs/general/contributor-guide.md

```diff
@@ -58,55 +54,83 @@ Good First Issue curates easy pickings from this project, and helps you make you
 - [Apache APISIX® Java Plugin Runner](https://github.com/apache/apisix-java-plugin-runner/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)
 - [Apache APISIX® Go Plugin Runner](https://github.com/apache/apisix-go-plugin-runner/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)
 
-### Create your branch
+## Git Flow
+
+To contribute code or documentation, setup your local machine for development.
 
-Switch to your forked master branch, pull codes from upstream, then create a new branch.
+### Fork the repo and clone your fork
+
+Fork the [Apache APISIX repo](https://github.com/apache/apisix/) and clone your fork to your local machine.
 
 ```sh
-$ git checkout master
-$ git pull upstream master
-$ git checkout -b IssueNo
+git clone https://github.com/your-username/apisix.git
 ```
 
-Notice: We will merge PR using squash, commit logs will be different form upstream if you use one older branch.
-
-### Coding
+### Add 'upstream' repo to list of remotes
 
-1. Please obey the Code of Conduct during the process of development and finish the check before submitting the pull request.
+```sh
+git remote add upstream https://github.com/apache/apisix.git
+```
 
-2. Then push codes to your fork repo.
+You can then verify the configured remotes.
 
 ```sh
-$ git add modified-file-names
-$ git commit -m 'commit message'
-$ git push origin issueNo
+git remote -v
 ```
 
-### Submit PR
+Learn more about Git and GitHub flow by following the [GitHub Docs](https://docs.github.com/en/get-started/quickstart/fork-a-repo).
 
-1. Send a pull request to the master branch.
+**Note**: Setup your username and email address to make sure that your ID shows up in the contributor list.
 
-2. The mentor will do codes review before discussing some details (including the design, the implementation and the performance) with you.
+```sh
+git config --global user.name "username"
+git config --global user.email "mail address"
```

Comment:
```suggestion
git config --global user.name "full name"
git config --global user.email "mail address"
```
It is a bad practice to use the GitHub username in the git config. You should always use your full name and the mail address tied to your GitHub account or any other service you might be using. It is the email that ties the commits to a developer not the username.